services/dbLogWriter/db_log_writer.go

```diff
@@ -50,6 +50,20 @@ func StartWritingLogs() {
 			if appRuntime.ShutdownRequested.Load() == true {
 				if len(channels.LogChannel) == 0 {
 					return
+				} else {
+					for i := 0; i < logBatchSizeStandard; i++ {
```

Comment:
This will hang the program! Think under which conditions.

Reply:
We need to send all the logs in the channel rather than in a batch of 100, so gotta replace batchSizeStandard with len(LogChannel) . Is it ?